.github/workflows/test_ert_with_slurm.yml

```diff
@@ -0,0 +1,76 @@
+on:
+  workflow_call:
+    inputs:
+      os:
+        type: string
+      python-version:
+        type: string
+jobs:
+  test-poly-on-slurm:
+    name: Run ert tests
+    timeout-minutes: 20
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - uses: actions/setup-python@v4
+      id: setup_python
+      with:
+        python-version: ${{ inputs.python-version }}
+        cache: "pip"
+        cache-dependency-path: |
+          setup.py
+          pyproject.toml
+
+    - name: Get wheels
+      uses: actions/download-artifact@v3
+      with:
+        name: ${{ inputs.os }} Python ${{ inputs.python-version }} wheel
+
+    - name: Install wheel
+      run: |
+        find . -name "*.whl" -exec pip install "{}[dev]" \;
+
+    - name: Install and setup slurm
+      run: |
+        set -e
+
+        sudo apt install slurmd slurmctld -y
+
+        cat <<EOF > slurm.conf
```

Comment:
Is this not missing a bracket ? `>`

Reply:
No, this is a bash "HERE" document; 

https://en.wikipedia.org/wiki/Here_document